### PR TITLE
[bug_5541] pep8

### PR DESCRIPTION
**Pull request by krzyhub**
https://bugzilla.stxnext.pl/show_bug.cgi?id=5541
https://gist.github.com/krzyhub/ef68701ee85893c8821e

### CODE REVIEW

File: src/presence_analyzer/main.py [hunk 1]
```diff
@@ -7,12 +7,10 @@
...
+    'runtime', 'data', 'sample_data.csv'
 )
 
 
 app = Flask(__name__)  # pylint: disable=invalid-name
-app.config.update(
-    DEBUG = True,
-    DATA_CSV = MAIN_DATA_CSV
-)
+app.config.update(DEBUG=True, DATA_CSV=MAIN_DATA_CSV)
```
Comment (mjelonek92) [inline src/presence_analyzer/main.py]: Please keep to this style: 
https://gist.github.com/krotkiewicz/3134c608c91cddbf21e5#file-functioncalls-py

**Comment by mjelonek92** [comment]
Attach results of pep8, pylint, tests (as a secret gist) to the description + a link to the bug on bugzilla.

Keep the style of title as follows please: ```[bug_XXXX] TITLE```

**Comment by dpolcyn** [comment]
Please squash commits.

**Comment by mjelonek92** [comment]
Looks good :+1:

**Pull request merged**